plugins/index.md

```diff
@@ -319,27 +297,15 @@ deb http://deb.theforeman.org/ plugins {{page.version}}
   </p>
 </div>
 
-<div class="installation_os installation_os_el6 installation_os_el7">
-  <p>
-    The naming of packages is as follows:
-  </p>
-
-  <ul>
-    <li>Packages for Foreman 1.10 or newer have a <code>tfm-rubygem-foreman_</code> codefix, while packages for Foreman 1.9 or older will have a <code>ruby193-rubygem-foreman_</code> codefix instead.  Adapt any instructions to suit.</li>
-    <li>Smart proxy packages have a <code>rubygem-smart_proxy_</code> codefix.</li>
-    <li>Hammer CLI packages for Foreman 1.10 or newer have a <code>tfm-rubygem-hammer_cli_</code> codefix, while CLI packages for 1.9 or older will have a <code>rubygem-hammer_cli_</code> codefix only.</li>
-  </ul>
-</div>
-
-<div class="installation_os installation_os_fedora19">
+<div class="installation_os installation_os_el7">
   <p>
     The naming of packages is as follows:
   </p>
 
   <ul>
-    <li>Foreman plugin packages have a <code>rubygem-foreman_</code> codefix.  Adapt any instructions to suit.</li>
+    <li>Packages for Foreman have a <code>tfm-rubygem-foreman_</code> codefix.</li>
     <li>Smart proxy packages have a <code>rubygem-smart_proxy_</code> codefix.</li>
-    <li>Hammer CLI packages have a <code>rubygem-hammer_cli_</code> codefix.</li>
+    <li>Hammer CLI packages have a <code>tfm-rubygem-hammer_cli_</code> codefix.</li>
```

Comment:
"codefix" should really be "prefix", I presume?

Reply:
`<code>` is OK, no? So a second commit with `s/codefix/prefix/g` would be nice :heart:

Reply:
Yeah, `<code>` is fine, just wanted to tease you ;)
Now thinking about it, given this is markdown, we can also use backticks, which are more readable IMHO. I'll add two commits when back at the laptop.

Reply:
Mhh, nope, the rest is HTML to, so let's keep it that way for now. Pushed the `s/codefix/prefix/g` change.